files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -664,6 +669,10 @@ audit-secureblue:
             status="$STATUS_FAILURE"
                 warnings+=("> $f has inter-process communications access!")
             fi
+            if hasPermission "$permissions" "device" "all"; then
+                [[ "$status" != "$STATUS_FAILURE" ]] && status="$STATUS_WARNING"
+                warnings+=("> $f has access to all devices!")
```

Comment:
Good point. I now switched it to "has device=all permission, granting access to GPU, input devices, virtualization, and shared memory".

Reply:
> "All devices" feels a little broad. Perhaps some explanation what this means? Like USBs, webcams, etc.?

It means unfiltered access to `/dev` which enables the tty sandbox escape:
https://codeberg.org/hanako/pentest-scripts/src/branch/main/ttyInjection.sh

Reply:
@RKNF404 fixed now, thanks.

@TheHooly do you think we should include that as well in the warning? I.e. has device=all permission, granting access to GPU, input devices, and virtualization, and introducing a vector for sandbox escapes"

Reply:
IMO: Could specify "raw USB" as one of the items as well, since that string would need at least 3 lines and this fits within this constraint. While the user can't do anything about this item until the next version of Flatpak comes arround (AFAICT), it still makes sense to inform the user about it. Otherwise LGTM, considering your own latest suggestion.

Reply:
> @RKNF404 fixed now, thanks.
> 
> @TheHooly do you think we should include that as well in the warning? I.e. "has device=all permission, granting access to GPU, input devices, and virtualization, and introducing a vector for sandbox escapes"

Personally, yes, I'm in favour of adding that to the warning.

Reply:
added raw USB and the sandbox escape warning.